packages/dx-chart-core/src/utils/series.js

```diff
@@ -134,9 +129,13 @@ export const createPieHitTester = createPointsEnumeratingHitTesterCreator(
     const halfAngle = Math.abs(startAngle - endAngle) / 2;
     const dx = px - x;
     const dy = py - y;
-    const r = Math.sqrt(dx * dx + dy * dy);
+    const r = getSegmentLength(dx, dy);
     const angle = mapAngleTod3(Math.atan2(dy, dx));
-    return getUniformDistance(r - rCenter, angle - angleCenter, halfRadius, halfAngle);
+    return Math.abs(r - rCenter) <= halfRadius && Math.abs(angle - angleCenter) <= halfAngle ? {
```

Comment:
This return code logic looks the same as in createBarHitTester()

Reply:
Resolved.